modules/introduction/pages/whats-new.adoc

```diff
@@ -1,193 +1,108 @@
+Couchbase Analytics Service introduces Remote Links which enable real-time operational analytics to obtain and analyze data from multiple Couchbase data clusters and datacenters in a separate cluster dedicated to the Analytics Service. To learn more about the feature, see xref:analytics:5_ddl.adoc#Remote_external_links[Remote Links and External Links].
 
-Couchbase transactions are architected to provide ACID guarantees at scale while ensuring there are no single points of failure or performance bottlenecks.
+[#whats-new-eventing-660]
+=== Eventing Service
 
-For more information, see xref:learn:data/transactions.adoc[Transactions].
+==== Timer Functionality
 
-==== SDK 3.0 Releases
+Couchbase Eventing Service introduces new Eventing timers that can be cancelled using the cancelTimer() function, or by creating a new timer with the same reference identifier as an existing timer. 
 
-Couchbase SDKs are bringing in major new features and updating to the latest interfaces for the platform. They support many new features available in this Server release while ensuring compatibility with older releases.
+Recurring timers are also fully supported, and can easily be used to create repetitive logic by using a timer callback to create fresh timers. 
 
-For more information, see the SDK 3 documentation.
+Timer scheduling enables timers to be created for days, weeks, or years in the future with no adverse performance impact. 
 
-https://docs.couchbase.com/c-sdk/3.0/hello-world/start-using-sdk.html[C SDK] | https://docs.couchbase.com/go-sdk/2.0/hello-world/start-using-sdk.html[Go SDK] | https://docs.couchbase.com/dotnet-sdk/3.0/hello-world/start-using-sdk.html[.NET SDK] | https://docs.couchbase.com/java-sdk/3.0/hello-world/start-using-sdk.html[Java SDK] | https://docs.couchbase.com/nodejs-sdk/3.0/hello-world/start-using-sdk.html[Node.js SDK] | https://docs.couchbase.com/php-sdk/3.0/hello-world/start-using-sdk.html[PHP SDK] | https://docs.couchbase.com/python-sdk/3.0/hello-world/start-using-sdk.html[Python SDK] | https://docs.couchbase.com/scala-sdk/1.0/start-using-sdk.html[Scala SDK]
+For detailed examples on how to use Eventing timers, see xref:eventing:eventing-examples-cancel-overwrite-timer.adoc[Cancel or Overwrite Timer] and xref:eventing:eventing-examples-recurring-timer.adoc[Recurring Timer].
 
+==== OnDelete Handler and UI Enhancements
 
-==== Query Enhancements
+The OnDelete Handler provides a new "options" parameter to indicate if a document was deleted or expired. 
 
-This release packs numerous additions to N1QL and the Query Service to provide a comprehensive query development toolkit for building applications.
+Key Eventing statistics in the UI are now co-located with each Functions lifecycle control. 
 
-===== Window Functions
+These enhancements simplify the effort and code required to make robust business logic.
```

Comment:
Will there be something to xref to?

Reply:
Will add later, once the eventing PR is merged